Engineering/Management/Pomodoro Technique.md

```diff
@@ -0,0 +1,29 @@
+  
+
+---
+
+tags: engineering, management, time
+
+author: Tran Hoang Nam
+
+---
+
+**Pomodoro Technique**
+
+  
+#### What is Pomodoro Technique?
+
+The Pomodoro Technique is a time-management philosophy that aims to give the user maximum focus and creative freshness, enabling them to finish projects more quickly and with less mental exhaustion.
+
+It is an easy process. You allocate time for each project throughout the day in little bursts and take breaks as needed. 25 minutes of labor are followed by a 5-minute rest.
+
+A "pomodoro" is a 25-minute work block, so named after the Italian word for tomato. The method's name comes from Francesco Cirillo, who utilized a kitchen timer in the form of a tomato as his personal timer. You take a 15-20 minute break after four "pomodoros" (100 minutes of work time and 15 minutes of break time) have been completed.
+
+You mark your progress with a "X" after each completed pomodoro and keep track of how many times throughout each 25-minute period you were tempted to delay or switch to working on anything else.
+
+#### Why does it useful?
+The Pomodoro technique is extremely efficient because it allows you to properly manage your time and work on a subject without interruptions. It is also advantageous since it assists you in becoming more disciplined and thinking about your task. This method is intended to reduce multitasking and boost concentration. Multitasking is far more difficult than it appears, and it leads to distraction and tiredness. The Pomodoro method is a simple and efficient strategy that helps you to focus on one job at a time.
```

Comment:
Although, there is nothing particularly wrong with this paragraph, it comes out as very general. For instance, you can replace the word "pomodoro" with any other technique, and it will most likely hold true.